src/bucket/BucketList.h

```diff
@@ -402,10 +402,24 @@ class BucketListDepth
     friend class testutil::BucketListDepthModifier;
 };
 
+struct EvictionMetrics
+{
+    // Evicted entry "age" is the delta between its liveUntilLedger and the
+    // ledger when the entry is actually evicted
+    uint64_t evictedEntriesAgeSum;
```

Comment:
uninitialized variables

Reply:
(also see my other comment regarding average age, I think you can remove this struct completely, and only keep evictionCycleStartLedger)